src/react/account/details/properties/AccountProperties.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+import type { Account } from '../../../../types/account';
+import AccountInfo from './AccountInfo';
+import AccountKeys from './AccountKeys';
+import { AutoSizer } from 'react-virtualized';
+import React from 'react';
+import styled from 'styled-components';
+
+type Props = {
+    account: Account,
+};
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    height: ${props => props.height}px;
+    width: ${props => props.width}px;
+`;
+
+function AccountProperties({ account }: Props) {
+    return (
+        <AutoSizer>
+            {({ height, width }) => (
+                <Container height={height} width={width}>
+                    <AccountInfo account={ account }/>
+                    <AccountKeys account={ account }/>
```

Comment:
We might want to move the account keys to its own **Keys** "account tab". I remember creating "Keys" component in `account/details`. 
That change should be quite easy and has two main advantages
- Separation of concern.
- We would have more space to display items.
- No need to load access keys on every account selection anymore.

Waiting for @terrydervaux approval

Reply:
Long story short, I want to keep root user access keys in the `Properties` tab.

We planned to have 7 tabs in the account page design. We want to use our "separation of concern capabilities" for account consumption metrics, storage locations, and IAM entities.

It's not a [best practice to have, keep, and use root user access keys](https://docs.aws.amazon.com/general/latest/gr/aws-security-audit-guide.html#aws-security-audit-review-account), our client should not have "a lot of keys". The table will be most of the time empty.

Tabs planned for account page:
- Metrics
- Users
- Groups
- Roles 
- Policies
- Storage location
- Properties

Reply:
In my opinion, most of these items should have their own page:
- Metrics
- Users
- Groups
- Roles
- Policies
- Storage location
- Properties